src/js/Content/Features/Community/Inventory/FInventoryMarketHelper.js

```diff
@@ -128,7 +123,7 @@ export default class FInventoryMarketHelper extends Feature {
          * or if not in own inventory but the item is marketable, build the HTML for showing info
          * TODO Fix the second condition: only add average price of three cards for booster packs in own inventory
          */
-        if ((ownsInventory && _restriction > 0 && !_marketable && !expired && hashName !== "753-Gems") || _marketable) {
+        if ((ownsInventory && restriction && !_marketable) || _marketable) {
```

Comment:
Be careful, if `restriction === 0` it is falsy.

Reply:
The value is a `bool` now, see changes at 11f6892, 863ce51.